easybuild/easyconfigs/a/argtable/argtable-2.13-intel-2018a.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'argtable'
+version = '2.13'
+
+homepage = 'http://argtable.sourceforge.net/'
+description = """ Argtable is an ANSI C library for parsing GNU style
+ command line options with a minimum of fuss. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s%s.tar.gz' % (name, version.replace('.', '-'))]
+checksums = ['8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf']
+
+sanity_check_paths = {
+    'files': ['lib/libargtable2.so', 'lib/libargtable2.la'],
```

Comment:
@hajgato Please use the shared library macro here

Reply:
@verdurin Done!